### DISCUSSION TITLE
Add ability to set shipping address for customer with Admin API

### DISCUSSION THREAD

**Discussion by winston0410** [medusajs/medusa#4753] [Feature Requests]
I want to add shipping addresses for `customer` with Admin API for development seeding. I have checked the doc but I cannot find route that can do that, can we have that added into Medusa Admin API?

For example, this is the add shipping address route in Storefront API: https://docs.medusajs.com/api/store#tag/Customers/operation/PostCustomersCustomerAddresses

But no equivalent exists for Admin API

**Comment by srindom** [comment]
You can do this through the draft order API

**Reply by winston0410** [reply]
Sorry I think for draft order it is for creating a draft order, and I can set shipping address to the order. But what I want is to set shipping addresses at the customer, not at an order. Is that doable with Admin API?

**Reply by jono-allen** [reply]
We had this issue and ended up exposing our own API todo this that was very similar to the store api but allowed to set the customer via a url parameter.

**Reply by winston0410** [reply]
> exposing our own API todo

Do you mean you have created a custom endpoint that uses customer services to insert a customer?
